contribs/gmf/src/controllers/abstractdesktop.js

```diff
@@ -129,6 +146,94 @@ gmf.AbstractDesktopController = function(config, $scope, $injector) {
       .getFeatureOverlay();
   this.drawFeatureLayer.setFeatures(ngeoFeatures);
 
+  var ngeoFeatureHelper = $injector.get('ngeoFeatureHelper');
+
+  /**
+   * @type {ol.layer.Vector}
+   * @export
+   */
+  this.editFeatureVectorLayer = new ol.layer.Vector({
+    source: new ol.source.Vector({
+      wrapX: false,
+      features: new ol.Collection()
+    }),
+    style: function(feature, resolution) {
```

Comment:
Something like this https://github.com/camptocamp/ngeo/commit/2ce5c9ea509e12f57f3f44a8ed322ca570e686c1 ?